src/lib/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .icr_tools import ICRTools
 from .channels import Channels
 from .analysis_keys import AnalysisKeys
+from .code_schemes import CodeSchemes
```

Comment:
This means you should be able to use from src.lib import CodeSchemes everywhere else instead from src.lib.code_schemes import CodeSchemes. Super minor, but makes those imports ever so slightly cleaner.

Reply:
updated thanks